packages/ui/package.json

```
@@ -28,5 +35,8 @@
     "@mui/material": "^5.5.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
+  },
+  "jest": {
+    "preset": "../config/jest/presets/jest-preset.js"
```

```suggestion
    "preset": "config/jest/presets"
```

And ensure `"config": "workspace:*"` is part of `devDependencies`.

---

@kilemensi 👍🏽